app/assets/stylesheets/components/_repo-list.scss

```diff
@@ -10,17 +10,44 @@
   background-color: darken($base-background-color, 5%);
   border-radius: $base-border-radius;
   box-shadow: 0 1px 0 lighten($base-background-color, 5%);
-  flex-basis: 18.75rem;
-  flex-grow: 1;
   margin-bottom: $base-spacing * 1.5;
-  margin-right: $base-spacing * 1.5;
+  min-width: 18rem;
   overflow: hidden;
   transition: all 0.25s ease-out;
+  width: 100%;
+
+  @include media(700px) {
```

Comment:
Should these media queries be variables? They may also be specific breakpoints so carry on if so!